index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="description" content="Felix Ouma is an experienced software developer" />
+    <title>Felix Ouma&apos;s Portfolio</title>
+    <link rel="stylesheet" href="assets/styles/style.css" />
+</head>
+<body>
+    <main>
+        <header class="flex-container">
+            <div class="logo">
+                <a href="#">Felix Ouma</a>
+            </div>
+
+            <nav>
+                <div>
+                    <img src="assets/images/Union.svg" alt="Hambuger Menu" />
+                </div>
+            </nav>
+        </header>
+        <section class="intro">
+            <div>
+                <h2 class="title">I’m Felix Glad to see you!</h2>
```

Comment:
- [ ] **[REQUIRED CHANGES]**

- The`<h2>` used here is supposed to be an `h1 tag` because this is the main heading of your website so it needs to be an `h1 tag` to improve accessibilty. Kindly, fix that.